.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Our [Code of Conduct](https://www.classicpress.net/democracy/#democracy-conduct) and [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions)
+Our code of conduct is simple: Be good to one another, take criticism well and follow the democratic process. When contributing to this repository, please note we have a [code of conduct](https://www.classicpress.net/democracy/#democracy-conduct) with some [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions). 
```

Comment:
Usually is better to not add a description of the code of conducts that can change and we need to update it.
Better to leave a link and keep it simple or could be misunderstood.
Take that as example now Mozilla is changing the text on over 1000 repository and how is difficult to manage for this single description.

Reply:
Also, the exceptions do not apply to the code of conduct itself, rather to the larger democratic process.

Reply:
How should we handle the exceptions? I mean stating it?

Reply:
there is the community team for it and should be written on those pages.
I think that is enough the text on the heading, the description is kind of repetitive.